fabtests/pytest/efa/test_cntr.py

```diff
@@ -0,0 +1,7 @@
+import pytest
```

Comment:
Some of these tests aren't changed at all, it feels a little bloated copying them all

Reply:
Yes, that is also comment from @shefty. I will try to address in new revision.

Reply:
In new revision, I used `from default.test_xxx import test_yyy` to import a test from default test sets. This removed duplication.